inference-engine/tests/functional/plugin/gpu/shared_tests_instances/low_precision_transformations/reshape_transformation.cpp

```diff
@@ -27,29 +25,31 @@ const std::vector<ReshapeTransformationParam> params = {
         { 1, 3, 32 },
         { 1, 3, 4, 8 },
         { 256ul, ngraph::Shape{ 1, 1, 1 }, { 0.f }, { 255.f }, { 0.f }, { 25.5f } },
-        true
+        "Reshape",
+        "U8"
     },
     // 4D -> 3D
     {
         { 1, 3, 16, 16 },
         { 1, 3, 256 },
         { 256ul, ngraph::Shape{ 1, 1, 1, 1 }, { 0.f }, { 255.f }, { 0.f }, { 25.5f } },
-        true
-    },
-    // 4D -> 3D
-    {
-        { 1, 3, 16, 16 },
-        { 0, 3, -1 },
-        { 256ul, ngraph::Shape{ 1, 3, 1, 1 }, { 0.f }, { 255.f }, { 0.f, 0.f, 0.f }, { 255.f, 25.5f, 2.55f } },
-        true
-    },
-    // 4D -> 2D
-    {
-        { 1, 3, 4, 8 },
-        { 1, -1 },
-        { 256ul, ngraph::Shape{ 1, 1, 1, 1 }, { 0.f }, { 255.f }, { 0.f }, { 25.5f } },
-        true
+        "Reshape",
+        "U8"
     },
+//    // 4D -> 3D
+//    {
+//        { 1, 3, 16, 16 },
+//        { 0, 3, -1 },
+//        { 256ul, ngraph::Shape{ 1, 3, 1, 1 }, { 0.f }, { 255.f }, { 0.f, 0.f, 0.f }, { 255.f, 25.5f, 2.55f } },
+//        true
+//    },
+//    // 4D -> 2D
+//    {
+//        { 1, 3, 4, 8 },
+//        { 1, -1 },
+//        { 256ul, ngraph::Shape{ 1, 1, 1, 1 }, { 0.f }, { 255.f }, { 0.f }, { 25.5f } },
+//        true
+//    },
```

Comment:
Could you uncomment and refactor this code?

Reply:
done